src/assets/stylesheets/index.scss

```diff
@@ -1156,7 +1178,7 @@ section:focus >,
 
 // -- general badges -- //
 
-.bg-middle-grey {
+.bagde.bg-middle-grey {
```

Comment:
* [ ] Weird typo. Is this unused? FYI I also see two `.jr-list-bagde` selectors in the existing CSS.